html/videoroomtest.js

```diff
@@ -162,7 +163,13 @@ $(document).ready(function() {
 											myid = msg["id"];
 											mypvtid = msg["private_id"];
 											Janus.log("Successfully joined room " + msg["room"] + " with ID " + myid);
-											publishOwnFeed(true);
+											if (subscriber_mode) {
+                                                						$('#videojoin').hide();
+                                                						$('#videos').removeClass('hide').show();
+                                            						}
+                                            						else
```

Comment:
Coding style: `else` should be on the same line as the closing bracket, and since it's after a bracket the other branch should be in brackets too (even if it has a single line of "action").